### ISSUE TITLE
Steel block to steel slab fix

### ISSUE THREAD

**Issue by b0bst3r** [redcreeper14385/BetterBees#1]
Issue: Bee's produce Mekanism Steel ingots, these steel ingots make Mekanism steel blocks, Mekanism steel blocks cannot make Immersive Engineering steel slabs.  

Side Issue: I haven't progressed far enough to discover if Mekanism steel blocks are recognised in Immersive Engineering multiblocks, so rather than insert a recipe just to do Mekanism Steel blocks > Immersive Engineering Slabs I changed the recipe 9x steel ingots to make IE steel block. Confirmed by AStranar in Discord that IE multiblocks (namely Arc Furnace and Excavator) do not form using Mekanism steel blocks.

Things to consider:  Steel bees require 1 steel block, this HAS to be a Mekanism steel block, you can still make these by inserting an Iron block into the IE blast furnace, you cannot make an IE Steel block in the Blast Furnace.

Please note you need Mekanism steel ingots for AE2 Inscriber Press crafting, so if you don't take my fix and work another then please look at the wider picture.

This fix means 3x3 grid of steel ingots makes the IE Steel block not the Mekanism steel block.

Add in line `craftingTable.removeRecipe(<item:mekanism:block_steel>);` to scripts\mekanism\a_remove_mekanism.zs

Add in section `
//Steel Block
craftingTable.addShaped("steel_block", <item:immersiveengineering:storage_steel>, [
    [<item:mekanism:ingot_steel>, <item:mekanism:ingot_steel>, <item:mekanism:ingot_steel>],
    [<item:mekanism:ingot_steel>, <item:mekanism:ingot_steel>, <item:mekanism:ingot_steel>],
    [<item:mekanism:ingot_steel>, <item:mekanism:ingot_steel>, <item:mekanism:ingot_steel>]
]);`
to scripts\immersiveengineering\add recipe.zs

So to recap - this fix allows for crafting of IE steel slabs, fixes IE steel blocks being craftable for IE Arc Furnace and Excavator, still allows Mekanism steel blocks to be made for bee flowers (Iron block in Blast Furnace) and allows Mekanism Steel ingots to exist for hard coded AE2 Press recipes and some other recipes I just found.

**Comment by b0bst3r** [comment]
Please note //Steel Block is a comment and a new should be after that before craftingTable.addShaped.............
